### PR TITLE
feat(modal-explorer): #MED-220 add modal to see app changes

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

Create modal on first used of users after the refont. 
The display management of the modal is done with preferences.

image
image
image

## Checklist tests

check the display and the good and the proper functioning of the modal display with different clicks (exit, later, etc.).

## Issue ticket number and link

[MED-220](https://jira.support-ent.fr/browse/MED-220)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/modal-explorer/ModalExplorer.tsx
Comment (niko-amosse) [inline frontend/src/components/modal-explorer/ModalExplorer.tsx:1]: ce composant est copié/coller d'edifice mais avec la possibilité de mettre nos images

File: frontend/src/hooks/useOnboardingModal.ts
Comment (niko-amosse) [inline frontend/src/hooks/useOnboardingModal.ts:1]: copié/collé d'edifice sans modif
c'est ici qu'est géré l'affichage de la modal avec les prefs

File: frontend/src/components/modal-explorer/ModalExplorer.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.med-modal-explorer-swiper {
+  height: 350px !important;
+  display: flex !important;
+  justify-content: center !important;
+  flex-direction: column !important;
+}
+.med-modal-explorer-image {
+  max-width: 580px;
+  max-height: 250px;
+}
```
Comment (Mattox0) [inline frontend/src/components/modal-explorer/ModalExplorer.scss:10]: t'as mis des px en dure, ton composant est responsive ?

File: frontend/src/components/modal-explorer/ModalExplorer.tsx
Comment (alicedraillard) [reply frontend/src/components/modal-explorer/ModalExplorer.tsx:1]: indique où les différences avec le composant Edifice sont

File: frontend/src/hooks/useOnboardingModal.ts
Comment (alicedraillard) [reply frontend/src/hooks/useOnboardingModal.ts:1]: idem

File: frontend/src/components/modal-explorer/ModalExplorer.scss (hunk 1 above)
Comment (niko-amosse) [reply frontend/src/components/modal-explorer/ModalExplorer.scss:10]: oui responsive ok

File: frontend/src/components/modal-explorer/ModalExplorer.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+              {items.map((item, index) => {
+                return (
+                  <SwiperSlide
+                    key={index}
+                    className="med-modal-explorer-swiper"
+                  >
+                    <Image
+                      className="mx-auto my-12 med-modal-explorer-image"
+                      loading="lazy"
+                      src={`${item.src}`}
```
Comment (niko-amosse) [inline frontend/src/components/modal-explorer/ModalExplorer.tsx:85]: la différence est ici, on met direct le chemin de l'image sans préfixe forcé comme chez edifice

File: frontend/src/hooks/useOnboardingModal.ts
Comment (niko-amosse) [reply frontend/src/hooks/useOnboardingModal.ts:1]: ici aucune modif, c'est copié/collé d'edifice

**Review by Mattox0 — APPROVED** [review]

File: frontend/src/components/modal-explorer/ModalExplorer.tsx
Comment (niko-amosse) [reply frontend/src/components/modal-explorer/ModalExplorer.tsx:1]: j'ai rajouté com

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**